docarray/array/storage/weaviate/backend.py

```diff
@@ -150,12 +150,9 @@ def _get_schema_by_name(self, cls_name: str) -> Dict:
 
     def _load_or_create_weaviate_schema(self):
         """Create a new weaviate schema for this :class:`DocumentArrayWeaviate` object
-        if not present in weaviate or if ``cls_name`` not provided, else if ``cls_name`` is provided
-        load the object with the given ``cls_name``
+        if ``self._config.name`` is None. If ``self._config.name`` is provided and
```

Comment:
please double check this line

Reply:
Sure, I have updated the docstring to make it more precise, thanks.